CMakeLists.txt

```diff
@@ -170,6 +170,10 @@ if(USE_TENSORRT)
     add_definitions(-DTVM_GRAPH_RUNTIME_TENSORRT)
     set(USE_TENSORRT OFF)
 endif()
+if(WITH_TENSORFLOW_LITE_LIB)
+    message("Adding TENSORFLOW_LITE_LIB to DLR shared library")
+    list(APPEND TVM_RUNTIME_LINKER_LIBS -llog -Wl,--whole-archive ${WITH_TENSORFLOW_LITE_LIB} -Wl,--no-whole-archive)
```

Comment:
is the second `-Wl` superfluous?

Reply:
@zhiics No, it is not, because we want apply `--whole-archive` option only to the TFLite lib and not to other libs.